.env

```diff
@@ -0,0 +1,2 @@
+PORT=3000
+DB_URL=mongodb+srv://user:CEUoCWpUQVgHrtWp@cluster0.73zeuje.mongodb.net/EPAM_NodeJS_DB?retryWrites=true&w=majority
```

Comment:
don't push sensitive data into GitHub

Reply:
I know .env should always be in .gitignore, but I decided to push it simply for the mentor's convenience - so he can test the API without the need to set up his own DB from scratch. I will make sure to delete it in the next commits.